core/store/models/job_spec.go

```diff
@@ -245,17 +245,17 @@ type Topics [][]common.Hash
 
 // Scan coerces the value returned from the data store to the proper data
 // in this instance.
-func (t Topics) Scan(value interface{}) error {
-	jsonStr, ok := value.(string)
-	if !ok {
+func (t *Topics) Scan(value interface{}) error {
```

Comment:
This was completely broken before. It left `Initiator.Topics` as nil always.

Reply:
Does this mean it wasn't really being used?

Reply:
It means `EthLog` most likely has not had working filtering based on Topics for some time.

I have to assume nobody complained because nobody is using it.

Reply:
See: https://github.com/smartcontractkit/chainlink/blob/develop/core/store/models/log_events.go#L141